Sprint5 AI ahmet

yildizahmetali opened the pull request:
Fixes #43 

**What was changed?**
Replaced Mock AI with normal AI

ruthvikm committed:
AI Module

Commit:
Merge remote-tracking branch 'origin/BubbleScan-AI-Module' into process_scantron_branch

ruthvikm committed:
returns JSON data

ruthvikm committed:
added init file

Commit:
replaced mock AI with our AI

rkarmuri committed:
"Removed node modules"

rkarmuri committed:
Merge branch 'main' into BubbleScan-AI-Module

rkarmuri committed:
removing duplicate AI folder

rkarmuri reviewed (commented):
There are several issues with your setup and code. Please address the below:

1. Get the latest data from the Bubble Scan AI module that returns the JSON data.
2. You have multiple Bubble scan AI modules, please remove the duplicates.
3. Remove your personal documents in the Server code instances folder.
4. As you can see in your output JSON data, the output is wrong and it is because the real AI module is using template.jpg for processing the PDF documents and your code doesn't read the template.jpg file to process. Please copy the template.jpg file to your Server code application folder.
5. You need to split the PDF processing and convert the JSON to CSV data in two different classes, right now you combined the 2 classes which is not accomplishing anything. You are only receiving the uploaded PDF but you are not connected to the AI module to receive the JSON data.
6. The AI module right now is dependent on the PDF file Super30 but you need to change it to accept any PDF from the React component.
Please address the above comments and update your pull request.

rkarmuri committed:
Replaced mock AI with real AI

yildizahmetali commented:
1- I got the latest data from the AI team
2- Removed the duplicates
3- Removed my personal items
4- I copied the template.jpg file to my Server code application folder
5- I split PDF processing
6- I changed the dependency

rkarmuri reviewed (dismissed):
With the new changes, everything looks good. Nice work.
